### PR TITLE
change Ituma Called-Station-Id to 'Location;SSID'

### PR DESCRIPTION
**Pull request by RoadRunnr**

### CODE REVIEW

**Comment by RoadRunnr** [comment]
mostly untested

**Review by vkatsuba — APPROVED** [review]

File: src/ergw_aaa_radius.erl [hunk 1]
```diff
@@ -315,6 +315,11 @@ system_time_to_universal_time(Time, TimeUnit) ->
     calendar:gregorian_seconds_to_datetime(Secs + ?SECONDS_FROM_0_TO_1970).
 -endif.
 
+format_eui(<<A:8, B:8, C:8, D:8, E:8, F:8>>) ->
+    io_lib:format("~2.16.0B-~2.16.0B-~2.16.0B-~2.16.0B-~2.16.0B-~2.16.0B", [A, B, C, D, E, F]);
+format_eui(MAC) ->
+    io_lib:format("~w", MAC).
```
Comment (vkatsuba) [inline src/ergw_aaa_radius.erl:321]: Looks like need wrap to list `MAC`: 
```erlang 
io_lib:format("~w", [MAC]).
```

**Review by vkatsuba — CHANGES_REQUESTED** [review]

File: src/ergw_aaa_radius.erl [hunk 2]
```diff
@@ -358,8 +363,9 @@ vendor_ituma('Wifi-Connect-Typ', #{'WLAN-Authentication-Mode' := open}, Attrs) -
     [{?'IM_Wifi_Connect_Type', <<"o">>}|Attrs];
 vendor_ituma('SSID', #{'SSID' := SSID}, Attrs) ->
     [{?'IM_SSID', SSID}|Attrs];
-vendor_ituma('Called-Station-Id', #{'SSID' := SSID, 'BSSID' := BSSID}, Attrs) ->
-    Id = lists:flatten(lists:join($;, [BSSID, SSID])),
+vendor_ituma('Called-Station-Id', #{'SSID' := SSID, 'Location-Id' := Location}, Attrs) ->
+    LocId = format_eui(<< (binary_to_integer(Location)):6/bytes >>),
```
Comment (vkatsuba) [inline src/ergw_aaa_radius.erl:367]: Judging by the data that is transmitted in `radius_SUITE.erl` the `Location` is integer. Or this wrong logic, or CT should be update

**Review by vkatsuba — CHANGES_REQUESTED** [review]

File: src/ergw_aaa_radius.erl (hunk 1 above)
Comment (vkatsuba) [inline src/ergw_aaa_radius.erl:321]:
```suggestion
    io_lib:format("~w", [MAC]).
```

**Review by vkatsuba — APPROVED** [review]

**Pull request merged**